pylon/pylon_fake.py

```
@@ -6,10 +6,13 @@
 import sys
 import time
 import pathlib
+import logging
```

Ruff

`logging` imported but unused

Remove unused import: `logging`

(F401)